modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -46,7 +45,7 @@ $ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namesp
 
 . If required, as the cluster administrator, perform any additional maintenance required on the node for the kernel module upgrade.
 +
-If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label value to the new `$moduleVersion` as set in the `Module`.
+If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>` label value to the new `$moduleVersion` as set in the `Module`.
```

Comment:
@cdvultur As per our chat, replaced periods and dashes with underscores in the user-replaceable values. If this looks good, please /lgtm. Thx!

Reply:
For step 4:

`$ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>-`

Hyphen at end is OK?

Reply:
Hyphen is ok.

Reply:
Mentioning the steps is very helpful to users. No plans on updating the steps, but if we do, will update this accordingly.